### ISSUE TITLE
Changing Properties after Initialization

### ISSUE THREAD

**Issue by rosskimes** [samvermette/SVSegmentedControl#1]
I have having some problems when changing the height and segmentPadding after the view has been placed.  I have to resize a bit in landscape, but the view is not changing at all.  I even tried calling setNeedsDisplay: on it.

Other than that, it is working great.  Keep up the good work.

**Comment by samvermette** [comment]
Indeed, I was aware of this but decided there wouldn't be a use case where you'd be changing the height/padding once the view is added. Totally forgot about orientation change. Will get this fixed shortly, thanks!

**Comment by rosskimes** [comment]
Awesome.  Thanks!
